src/pytorch_lightning/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [1.7.6] - 2022-09-13
+
+### Fixed
+
+
+
```

Comment:
left over. we need to add this part in the next release. Not in this one.

Reply:
done.